sr/robot/sim_robot.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import division
```

Comment:
is this needed any more?

Reply:
Yes. The intent here is to add support for Python 3, not to move entirely to it.
These imports are needed for Python 2 (see also the commit message of https://github.com/srobo/sr-turtle/pull/3/commits/8a364cb0efdbb5f3b8c2789e1b618e3db2cef7a8).